tests/test_filters.py

```diff
@@ -871,6 +872,51 @@ def test_decimate_boundary():
     assert boundary.n_points
 
 
+def test_extract_surface():
+    # create a single hexahedral cell
```

Comment:
Again redundancy for the sake of clarity might be better
```suggestion
    # create a single quadratic hexahedral cell
```